awscli/clidriver.py

```diff
@@ -656,10 +657,15 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             value is returned.
 
         """
+        if hasattr(parsed_globals, 'read_timeout'):
```

Comment:
Interesting. Let's figure this out. If the if statement is not needed (only needed to make the test pass) let's remove it, keep it otherwise.